docs/management/advanced-options.asciidoc

```diff
@@ -46,6 +46,7 @@ adapt to the interval between measurements. Keys are http://en.wikipedia.org/wik
 `dateFormat:tz`:: The timezone that Kibana uses. The default value of `Browser` uses the timezone detected by the browser.
 `dateNanosFormat`:: The format to use for displaying https://momentjs.com/docs/#/displaying/format/[pretty formatted dates] of {ref}/date_nanos.html[Elasticsearch date_nanos type].
 `defaultIndex`:: The index to access if no index is set. The default is `null`.
+`defaultRoute`:: This setting specifies the default route when opening Kibana. You can use this setting to modify the landing page when opening Kibana.
```

Comment:
I'm wondering if we might include text about spaces here:

- The default route when opening Kibana. Use this setting to configure the landing page on a per-space basis.

- The default route when opening Kibana.  Use this setting to route users to a specific dashboard, application, or saved object as they enter each space.

Should we add something about this to the Spaces documentation page?  Something like:

Space-level landing page

You can create a custom experience for users by configuring the Kibana landing page on a per-space basis.  <explain why you would want to do this> To configure the landing page, use the `default.route` in Management > Advanced settings.

Reply:
> I'm wondering if we might include text about spaces here:

I like your second option here, I'll include this text.

> Should we add something about this to the Spaces documentation page?

I'll put something together for you to look at!